Author: Andreas Hoenen <andreas@hoenen-terstappen.de>
Description: Hotfix for BTS report #904675:
Treat legalnotice blocks like other blocks: escape # characters in url attributes.
--- a/xsl/xref.xsl
+++ b/xsl/xref.xsl
@@ -225,7 +225,8 @@
     <xsl:with-param name="string">
       <xsl:choose>
       <xsl:when test="$escape != 0 or ancestor::entry or ancestor::revision or
-                      ancestor::footnote or ancestor::term">
+                      ancestor::footnote or ancestor::term or
+                      ancestor::legalnotice">
         <xsl:call-template name="string-replace">
           <xsl:with-param name="string">
             <xsl:call-template name="string-replace">
